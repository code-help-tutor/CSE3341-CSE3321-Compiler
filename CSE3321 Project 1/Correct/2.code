
array int X , Y , XY 	; begin

          X = 0;
		  
		  new[10] X;

          end  
      