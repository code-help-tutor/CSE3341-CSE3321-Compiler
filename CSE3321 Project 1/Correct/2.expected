ARRAY
ID[int]
ID[X]
COMMA
ID[Y]
COMMA
ID[XY]
SEMICOLON
BEGIN
ID[X]
EQUAL
CONST[0]
SEMICOLON
NEW
LBRACE
CONST[10]
RBRACE
ID[X]
SEMICOLON
END
